src/audiovideocontroller/DefaultAudioVideoController.ts

```diff
@@ -1148,4 +1172,11 @@ export default class DefaultAudioVideoController
       );
     });
   }
+
+  private useUpdateTransceiverController(): boolean {
+    return (
+      this.meetingSessionContext.browserBehavior.requiresUnifiedPlan() &&
+      !!this.meetingSessionContext.videoUplinkBandwidthPolicy.updateTransceiverController
+    );
+  }
```

Comment:
If the policy never changes during a call, and neither does the browser behavior object, can we just set this as a private field during start rather than having it as a dynamic method?

Reply:
Good callout. Updated.